api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -195,6 +195,7 @@ def from_orm(cls: Any, offer: Offer):  # type: ignore
     externalTicketOfficeUrl: Optional[str]
     extraData: Optional[OfferExtraData]
     isExpired: bool
+    is_forbidden_to_underage: bool
```

Comment:
tu ne le mets pas en camel case comme les autres ?

Reply:
Il y a cette ligne qui va le transformer :) https://github.com/pass-culture/pass-culture-main/blob/pc-11880-offer-is-forbidden/api/src/pcapi/routes/native/v1/serialization/offers.py#L213